### DISCUSSION TITLE
Options Picker with data provider and selecting ‚choose an option‘ throws error

### DISCUSSION THREAD

**Discussion by herzkerl** [Budibase/budibase#10054] [Help] [answered]
Dear community,  I’ve created a form with an options picker using a data provider as described in [your documentation](https://docs.budibase.com/docs/option-picker#using-a-data-provider).

It works like in the video except for one caveat: When _un_selecting (i.e. selecting ‚Choose an option‘) it shows an error message ‚required‘. The filtering works, so all entries are shown, but in the video there’s no such error message—what am I doing wrong?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @herzkerl,

I think i know why this is happening. Is the field you are searching on required in the data section? (Screenshot attached below.)

<img width="787" alt="Screenshot 2023-03-16 at 12 49 41" src="https://user-images.githubusercontent.com/126772285/225622512-9a2779f6-4b82-4a77-b8f4-e32024769627.png">

There are 2 different ways to add form validations and the required element on the column's themselves is one method. To prevent this from happening all you need to do is un-toggle the required light-swtich in the screenshot above.

Hopefully this helps!
